lib/apiservers/engine/backends/container.go

```diff
@@ -47,27 +45,26 @@ import (
 	"github.com/docker/docker/utils"
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/go-units"
-	"github.com/docker/libnetwork/iptables"
-	"github.com/docker/libnetwork/portallocator"
-	"github.com/vishvananda/netlink"
 
 	"github.com/vmware/vic/lib/apiservers/engine/backends/cache"
 	viccontainer "github.com/vmware/vic/lib/apiservers/engine/backends/container"
 	"github.com/vmware/vic/lib/apiservers/engine/backends/convert"
 	"github.com/vmware/vic/lib/apiservers/engine/backends/filter"
-	"github.com/vmware/vic/lib/apiservers/engine/backends/portmap"
+	engerr "github.com/vmware/vic/lib/apiservers/engine/errors"
```

Comment:
please use errors package name if there is not name conflict.

Reply:
No one aliases a package unless there is a conflict.  You can do a search for errors and see there are conflicts.

Reply:
@sflxn it may not be the case here but I've found plenty of instances of aliases with no conflicts. It was very confusing.

Reply:
Yah, I see it bellow. Well, that import is basically useless. It is used in one place and does basically nothing. Feel free to replace it with fmt.Errorf